### PR TITLE
data: ShanmukhiKairuppala

### PR DESCRIPTION
**Pull request by ShanmukhiKairuppala**
added ShanmukhiKairuppala to link free 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2713"></a>

### CODE REVIEW

File: data/ShanmukhiKairuppala.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Shanmukhi Kairuppala",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": " I'm an Open Source enthusiast and an aesthete!",
+  "avatar": "https://github.com/ShanmukhiKairuppala.png",
+  "links": [
```
Comment (dan-mba) [inline data/ShanmukhiKairuppala.json:7]: You need commas between each object in the links array

Comment (krupalitrivedi) [reply data/ShanmukhiKairuppala.json:7]: That is correct, I will mention below the places where the comma (,) has to be added!

File: data/ShanmukhiKairuppala.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Sirius_2003",
+      "icon": "FaTwitter"
+    }
```
Comment (krupalitrivedi) [inline data/ShanmukhiKairuppala.json:12]: Consider adding a comma here, as one of your links end here

File: data/ShanmukhiKairuppala.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+     {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/shanmukhi-k-56535525b/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/shanmukhi_kairuppala/?next=%2F",
+      "icon": "FaInstagram"
+    }
```
Comment (krupalitrivedi) [inline data/ShanmukhiKairuppala.json:22]: Consider adding a comma here, as one of your links end here!

**Review by krupalitrivedi — CHANGES_REQUESTED** [review]
There are changes mentioned in the review!

Just make the changes in your fork and commit them, and the PR will be updated!

Let me know if you need any help!

File: data/ShanmukhiKairuppala.json (hunk 3 above)
Comment (ShanmukhiKairuppala) [reply data/ShanmukhiKairuppala.json:22]: Where can I make changes ?

Comment (ShanmukhiKairuppala) [reply data/ShanmukhiKairuppala.json]: got it. Thankyou !

**Review by dan-mba — APPROVED** [review]
Looks good to me.

**Pull request merged**